api/provider/aws/task_list.go

```diff
@@ -0,0 +1,75 @@
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go/service/ecs"
+	"github.com/quintilesims/layer0/common/models"
+)
+
+func (t *TaskProvider) List() ([]*models.TaskSummary, error) {
+	clusterNames, err := listClusterNames(t.AWS.ECS, t.Config.Instance())
+
+	if err != nil {
+		return nil, err
+	}
+
+	summaries := []*models.TaskSummary{}
+	for _, clusterName := range clusterNames {
+		runningTaskARNs, err := t.listTasksByStatus(clusterName, "RUNNING")
+		stoppedTaskARNs, err := t.listTasksByStatus(clusterName, "STOPPED")
```

Comment:
Instead of using a string, we could use the constants provided in aws-sdk-go: https://github.com/aws/aws-sdk-go/blob/master/service/ecs/api.go#L9159

`stoppedTaskARNs, err := t.listTasksByStatus(clusterName, ecs.DesiredStatusStopped)`